### PR TITLE
Add reflex cli tool.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

**Comment by dlorenc** [comment]
Update check seems unrelated: https://github.com/wolfi-dev/os/pull/4466

File: reflex.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - uses: git-checkout
+    with:
+      repository: https://github.com/reflex-dev/reflex
+      tag: v${{package.version}}
+      expected-commit: 9f7a7a1e4a38b1ab443451d1bbdc802c00a10739
+
+  - runs: |
+      poetry build
+      pip install --root ${{targets.destdir}} dist/reflex-*.whl
+      find ${{targets.destdir}}/usr/lib/python*/site-packages/ \( -type d -a -name test -o -name tests \) -o \( -type f -a -name '*.pyc' -o -name '*.pyo' \) -exec rm -rf '{}' \+
```
Comment (imjasonh) [inline reflex.yaml:40]: This seems like a useful shared pipeline 🤔

**Review by imjasonh — APPROVED** [review]

File: reflex.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - python3
+      # Reflex shells out to these these tools during init
+      - bash
+      - wget
+      - curl
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
```
Comment (vaikas) [inline reflex.yaml:19]: super nit, wonder if we should strive for alphabetizing packages so it's easy to see what's in there. Maybe it's just me.

Comment (imjasonh) [reply reflex.yaml:19]: That seems like a useful thing to add to https://github.com/chainguard-dev/yam and apply across this repo, if you're feeling so inclined.

Personally I'd love to take advantage of our knowledge of the dep graph to trim away redundant deps. 🙏

**Pull request merged**